### PR TITLE
[FR] Add "Article styling criteria/Writing"

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page "Article styling criteria/Writing".

I don't think it's necessary to translate it, since it concerns the creation of articles in English, but I did it anyway, trying to adapt it for the French.

### CODE REVIEW

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+Éviter :  Les BNs peuvent nommer votre beatmapset.
+
+Préférer : Les nominateurs peuvent nommer votre beatmapset.
+```
+
+## Perspective
+
+Les termes `joueur`, `utilisateur`, `skinner`, `scénariste` et `créateur` doivent être utilisés pour désigner le lecteur ou une autre personne. `ils`, `les`, `leur`, et `leurs` peuvent être utilisés si nécessaire.
+
+Les expressions `vous` et `votre` sont à éviter. Le `je` ne doit pas être utilisé. Les termes `nous`, `il` ou `sa` et `elle` ou `son` ne doivent pas être utilisés (voir ci-dessus pour les autres termes).
```
Comment (Satsukel) [inline wiki/Article_styling_criteria/Writing/fr.md:297]: ça serait pas plus "tu" et "ton" qui ne peuvent pas être utilisés ?

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+---
+
+[Les articles avec registres informels](#registre-des-langues) peut ignorer cette section ; cependant, il faut éviter de mettre des `I` dans les paragraphes.
+
+## Grammar and syntax
+
+Les articles doivent utiliser de préférence une grammaire et une [syntaxe américaines plus simples](https://www.thepunctuationguide.com/british-versus-american-style.html "Le Guide de la ponctuation").
```
Comment (Satsukel) [inline wiki/Article_styling_criteria/Writing/fr.md:305]:
```suggestion
Les articles doivent utiliser de préférence une grammaire et une [syntaxe américaines plus simple](https://www.thepunctuationguide.com/british-versus-american-style.html "Le Guide de la ponctuation").
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

File: wiki/Article_styling_criteria/Writing/fr.md (hunk 1 above)
Comment (DoryanR) [reply wiki/Article_styling_criteria/Writing/fr.md]: Ouais effectivement, j'ai modifier

**Review by Satsukel — APPROVED** [review]

**Comment by TicClick** [comment]
please adapt your translation to 596efd8dff27df4ce19401c51c3f809562ea6e95

> I did it anyway, trying to adapt it for the French.

that is the spirit -- I've done the same for its Russian version

**Review by TicClick — APPROVED** [review]

**Pull request merged**